package.json

```diff
@@ -138,6 +138,7 @@
     "front-matter": "^4.0.2",
     "hastscript": "^7.0.2",
     "hoist-non-react-statics": "^3.0.1",
+    "io-react-native-secure-storage": "^0.1.1",
```

Comment:
Can we remove this since we are not using it? It also needs a bit of configuration for Android which is not implemented in this PR.